README.md

```diff
@@ -31,3 +31,20 @@ cp .env.example .env.development
 ```
 
 ## Feel free to contribute :D
+
+### Run the project locally
+
+1. Fork the repository to get your copy on your account.
+2. Clone your forked repository on your machine.
+3. Open the cloned `frontend.mu` directory in your code editor.
+4. Run `pnpm install` in your terminal.
+5. Run `cd packages/frontendmu-nuxt`.
+6. Run `pnpm run dev`.
```

Comment:
@n-d-r-d-g instead of line 5 & 6 you can combine them into
5. Run `pnpm run nuxt dev`

pnpm is aware of the packages and will run that command in the nuxt package

Reply:
That's neat. Thanks for the suggestion!